config/hsc/processCcd.py

```diff
@@ -9,8 +9,8 @@
 hscConfigDir = os.path.join(getPackageDir("obs_subaru"), "config", "hsc")
 config.load(os.path.join(hscConfigDir, 'isr.py'))
 config.calibrate.photoCal.colorterms.load(os.path.join(hscConfigDir, 'colorterms.py'))
-config.charImage.measurePsf.starSelector.widthMin=0.9
-config.charImage.measurePsf.starSelector.fluxMin=4000
+config.charImage.measurePsf.starSelector["objectSize"].widthMin=0.9
```

Comment:
Yes, spaces around equals signs please, even in config files.  See the rest of the file.